packages/client/hmi-client/src/components/workflow/scenario-templates/base-scenario.ts

```diff
@@ -1,9 +1,21 @@
 import { Workflow } from '@/types/workflow';
 
+interface ScenarioHeader {
+	title: string;
+	question?: string;
+	description?: string;
+	examples?: string[];
+}
 export abstract class BaseScenario {
-	public static templateId: string;
+	public static templateId: string = '';
+
+	public static templateName: string = '';
+
+	public static header: ScenarioHeader = {
+		title: ''
+	};
```

Comment:
I am not entire sure what you are going for here, the inheritance and static declaration seem to in opposition to each other.

Reply:
I've moved these out of the class and into the component, which should be easier.
Ex.
```
const header: ScenarioHeader = Object.freeze({
	title: 'Situational awareness template',
	question: "What's likely to happen next?",
	description:
		'Calibrates the model to historical data to obtain the best estimate of parameters for the present, then forecasts into the near future.',
	examples: [
		'Anticipate the arrival of a new variants.',
		'Evaluate the potential impact of growing vaccine hesitancy and declining NPIs.'
	]
});
```